code/modules/clothing/rogueclothes/gloves.dm

```diff
@@ -26,6 +26,20 @@
 	drop_sound = 'sound/foley/dropsound/cloth_drop.ogg'
 	salvage_result = /obj/item/natural/hide
 
+/obj/item/clothing/gloves/roguetown/leatherb
+	name = "leather gloves"
+	desc = "Leather gloves usually worn by laborers. dyed black."
+	icon_state = "leather_gloves"
+	armor = list("blunt" = 30, "slash" = 10, "stab" = 20, "bullet" = 1, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	prevent_crits = list(BCLASS_CUT, BCLASS_STAB, BCLASS_BLUNT)
+	resistance_flags = null
+	blocksound = SOFTHIT
+	blade_dulling = DULLING_BASHCHOP
+	break_sound = 'sound/foley/cloth_rip.ogg'
+	drop_sound = 'sound/foley/dropsound/cloth_drop.ogg'
+	color = "#151615"
+	salvage_result = /obj/item/natural/hide
+
```

Comment:
```suggestion
/obj/item/clothing/gloves/roguetown/leather/black
	color = "#151615"
```

Reply:
You'll need to adjust the loadouts that get this too- From a cursory examination you only changed these two vars, but it's possible I missed something!

Reply:
I mean, if you change the typepath of these to be a subtype of regular leather gloves you'll need to adjust hte doctors loadout.